tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -142,6 +142,31 @@ public function testRemove()
         $this->assertEquals('defaultValue', $default);
     }
 
+    public function testRemoveValue()
```

Comment:
The test method has two cases. Could you split it into test method and data-provider?